web/src/elements/sidebar/SidebarItems.css.ts

```diff
@@ -0,0 +1,86 @@
+import { css } from "lit";
+
+import PFNav from "@patternfly/patternfly/components/Nav/nav.css";
+import PFPage from "@patternfly/patternfly/components/Page/page.css";
+import PFBase from "@patternfly/patternfly/patternfly-base.css";
+
+export const sidebarItemStyles = [
+    PFBase,
+    PFPage,
+    PFNav,
+    css`
```

Comment:
There's a lot of custom CSS here, but none of it is particularly locked-in.  All of my customized additions use the `--pf-global` namespace for their values, in the hopes that we can keep using those without fear.